### PR TITLE
Developed .html files and tests for method element(final By locator).

### PR DESCRIPTION
**Pull request by EreOo**
issue #12 
test for method element(final By locator).

### CODE REVIEW

File: src/test/java/com/wiley/autotest/framework/pages/TestElementPage.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    public TestElementPage checkElementBecomeVisible() {
+        element(By.id("waitElement")).should().beDisplayed();
+        return this;
+    }
+
+    public TestElementPage checkAnimationElement() {
+        element(By.id("myAnimation")).should().beDisplayed();
+        element(By.id("myAnimation")).should().beDisplayed();
+        element(By.id("myAnimation")).should().beDisplayed();
```
Comment (vefimofff) [inline src/test/java/com/wiley/autotest/framework/pages/TestElementPage.java]: Not sure i understand the idea of this particular test? What does it really check and in what situation do you expect it to fail? I think the best way would be to provide a comment to this method - explaining the idea of it

File: src/test/java/com/wiley/autotest/framework/tests/elements/NonexistentElement.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+/**
+ * Created by shekhavtsov on 21/07/2017.
+ */
+public class NonexistentElement extends BaseTest {
+
+    @Autowired
+    private TestElementPage testElementPage;
+
+    @Test(expectedExceptions = Exception.class)
```
Comment (vefimofff) [inline src/test/java/com/wiley/autotest/framework/tests/elements/NonexistentElement.java]: please specify what exactly exception is expected as Exception is too general

File: src/test/java/com/wiley/autotest/framework/tests/elements/NonexistentElement.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import com.wiley.autotest.framework.config.BaseTest;
+import com.wiley.autotest.framework.pages.TestElementPage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testng.annotations.Test;
+
+/**
+ * Created by shekhavtsov on 21/07/2017.
+ */
+public class NonexistentElement extends BaseTest {
```
Comment (vefimofff) [inline src/test/java/com/wiley/autotest/framework/tests/elements/NonexistentElement.java]: Lets add "ThrowsException" to a class name to make it more friendly

**Comment by EreOo** [comment]
@vefimofff I fixed and pushed.

**Pull request merged**